bin/cli-new.js

```
@@ -0,0 +1,704 @@
+  .argument('[action]', 'Action to perform (status, setup, login)', 'status')
+  .action(async (action) => {
+    switch (action) {
+      case 'status':
+        showAuthStatus();
+        break;
+      case 'setup':
+      case 'login':
+        console.log(chalk.yellow('\n🚧 Interactive auth setup coming soon!\n'));
+        console.log('For now, please use one of these methods:\n');
+        const { showAuthHelp } = await import('../lib/auth.js');
+        showAuthHelp();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)